hypertrace-core-graphql-attribute-store/src/main/java/org/hypertrace/core/graphql/attributes/AttributeModelTranslator.java

```diff
@@ -57,7 +80,24 @@ public Optional<AttributeModel> translate(AttributeMetadata attributeMetadata) {
     }
   }
 
-  @SuppressWarnings("unused")
+  public AttributeMetadata translate(final AttributeModel attributeMetadata) {
```

Comment:
So the question here is really - do we want to fix the API (which we _just_ merged and unfortunately I completely missed on review) or accept the tech debt? If we want to accept the tech debt, I would probably clarify the name here that this is for use on creation only. If we want to fix the API before we start using it, we'd change the create API to not accept an attributemetadata list, but a list of protos with only the fields supported on creation.

Reply:
Ah - I just looked again, we didn't add the create API, it was pre-existing. That explains the miss - we just exposed the old API that asks for an ID on create.

Given that, we would want to double check that the create API actually does generate the ID and doesn't assume it will be provided. If it does, we can go with just the quick rename.

Reply:
Got rid of this altogether by directly passing List<AttributeMetadata> to the `create` method.

Reply:
> Given that, we would want to double check that the create API actually does generate the ID and doesn't assume it will be provided.

Yes, verified it end-to-end and it is working.